crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -51,18 +51,40 @@ def start
       # the shell Process will be spawned in the background and therefore has
       # not a direct return value which we can use here
       if Crowbar::Installer.installing?
-        flash[:notice] = I18n.t(".installers.start.installation_ongoing")
+        msg = I18n.t(".installers.start.installation_ongoing")
+
+        respond_to do |format|
+          format.json {
+            head :im_used
+          }
+          format.html {
+            flash[:notice] = msg
+            redirect_to installer_url
+          }
+        end
       else
         ret = Crowbar::Installer.install
         case ret[:status]
         when 501
-          flash[:alert] = ret[:msg]
+          respond_to do |format|
+            format.json {
+              head :not_implemented
+            }
+          end
+          format.html {
+            flash[:alert] = ret[:msg]
+            redirect_to installer_url
+          }
         end
-      end
 
-      respond_to do |format|
-        format.json { head :ok }
-        format.html { redirect_to installer_url }
+        respond_to do |format|
+          format.json {
+            head :ok
+          }
+          format.html {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)